tmexp/metrics.py

```diff
@@ -63,18 +78,12 @@ def compute_metrics(bow_name: str, exp_name: str, force: bool, log_level: str) -
     num_topics, num_words = wordtopic.shape
     logger.info("Loaded, found %d words and %d topics.", num_words, num_topics)
 
-    similarity = np.zeros((num_topics, num_topics))
-    logger.info("Computing similarity between topics ...")
-    for i in range(num_topics):
-        dist_i = wordtopic[i, :]
-        for j in range(num_topics):
-            dist_j = wordtopic[j, :]
-            similarity[i, j] = np.sum(dist_i * np.log(dist_i / dist_j))
-    similarity = (similarity + similarity.T) / 2
-    logger.info("Computed similarity between topics.")
+    logger.info("Computing distinctness between topics ...")
+    distinctness = compute_distinctness(wordtopic, num_topics, num_words)
+    logger.info("Computed distinctness between topics.")
 
-    logger.info("Mean similarity per topic:")
-    metric_stats(np.sum(similarity, axis=1) / (num_topics - 1), 1, logger)
+    logger.info("Mean distinctness per topic:")
+    metric_stats(np.sum(distinctness, axis=1) / (num_topics - 1), 1, logger)
```

Comment:
```suggestion
    metric_stats(np.mean(distinctness, axis=1), 1, logger)
```

Reply:
yeah. we were dividing by `num_topics - 1` instead of `num_topics`, but it doesnt really matter I guess

Reply:
Right, better to keep the current version then (same for the other comment).

Reply:
as you want